README.md

```diff
@@ -275,20 +619,68 @@ We can discuss what is automated, what is not, and how that can be improved. Mor
 what are the production services and software offered by HPC, and can we trace their entire pipeline
 from source code to production deployment? Is there room for improvement?
 
+> there is a lot of discussion aroudn software stacks like this right now -- we should probably
+> sync up on it. There are more reasons than just RSE-ops to have a common software stack for HPC
+> at this level.  It also helps us push on vendors.
+
 ## Summary of Opportunities
 
 The following areas of RSE-ops are not well developed, and we have opportunity to build or seek out tools to fill the space.
 
 - **Application Development**: Development framework for interactive (browser based) applications for HPC. This relies on being able to ensure secure access in a browser.
+
+> I think people are calling this "cloud services" or "web applications". If you say "app
+> development in HPC people think simulation.
+
 - **Automated Scaling**:  Being able to scale a job, task, or service without thinking about it.
 - **On Demand Resources**: An HPC user should theoretically be able to request a custom resource, and "throw it away" when done. For HPC an administrative team would still need to monitor and maintain the resources, but to the user there could be a more interactive, customized experience. E.g., "I want an instance with this amount of memory, storage, and with this software installed. And I want to customize and request it in a web interface."
+
+> confused about this one because this is what SLURM and Flux do. You can ask for these things, and
```

Comment:
I mean more like "I want this, exactly, now." With SLURM and Flux (well SLURM I haven't used Flux) I have to wait 24 hours and then it poops and fails and I'm waiting again. I'm not just given the raw resource to interact with, when I need it, I'm put in a queue for something automated to be attempted once, and too bad if it fails. It's different.

Reply:
The wait time depends on how utilized the machine is -- most HPC machines are highly utilize.  There are debug queues for this, but they are not for persistent services like clouds.  Clouds have a lot more free resources -- and you also pay for the free resources.

My point here is that the paragraph is going to confuse people b/c resource managers do exactly what you are saying they don't.  I think the text should focus on how batch systems are not designed to support on-demand computing -- they aren't responsive; they're designed for things that take a long time and they don't even do lots of small jobs well.  It's not that they utilize resources inefficiently -- they actually have higher utilization than your typical cloud.

Kate Keahy has done a lot of work on this at ANL: https://scholar.google.com/citations?user=ocvPDcsAAAAJ  on the distinction bt/w HPC and infrastructure workloads and how you could combine them.  To some extent Flux is about this as well though their focus right now is on bridging Flux and Kubernetes, where k8s would handle the infra side.

Reply:
will integrate this!